app/views/proposals/detail/_observer.html.haml

```diff
@@ -0,0 +1,23 @@
+.card-for-observers
+  .row
+    .medium-12.column
+      .card
+        .card-head.column
+          %h2
+            Observers
+        .content-content.column
+          %ul.observer-list
+            - SubscriberList.new(proposal).triples.each do |user, role_str, observation|
```

Comment:
thoughts on assigning `SubscriberList.new(proposal).triples` to an instance var in the controller? would be in the spirit of "less ruby in the views"